modules/trusted-firmware-m/Kconfig

```diff
@@ -207,4 +207,13 @@ config ROM_START_OFFSET
 
 endif # !TFM_BL2
 
+config TFM_FLASH_MERGED_BINARY
+	bool "Combine with the TF-M and BL2 binaries when flashing the application"
```

Comment:
"Merge sample hex file with the TF-M ..." ?
First of all I think we should use the term "merge" as that is used elsewhere for this.
Secondly IMO it is not clear what is merged with TF-M and BL2 the way it is phrased.

Reply:
OK, agreed, the "merged" needs to be present in the prompt. Will rework this, thanks.

Reply:
`	bool "Flash a merged binary consisting of BL2 (optionally), TF-M, and application"
`

Reply:
would that make it more clear @hakonfam ?

Reply:
Yes!